src/pages/layer-2/networks.tsx

```diff
@@ -0,0 +1,212 @@
+    ["growThePieBlockspaceData", fetchGrowThePieBlockspace],
+    ["growThePieMasterData", fetchGrowThePieMaster],
+    ["l2beatData", fetchL2beat],
+  ],
+  REVALIDATE_TIME * 1000
+)
+
+export const getStaticProps = (async ({ locale }) => {
+  const [
+    ethereumTVLData,
+    growThePieData,
+    growThePieBlockspaceData,
+    growThePieMasterData,
+    l2beatData,
+  ] = await loadData()
+
+  const lastDeployDate = getLastDeployDate()
+  const lastDeployLocaleTimestamp = getLocaleTimestamp(
+    locale as Lang,
+    lastDeployDate
+  )
+
+  const requiredNamespaces = getRequiredNamespacesForPage("/layer-2/networks")
+
+  const contentNotTranslated = !existsNamespace(locale!, requiredNamespaces[2])
+
+  const layer2DataCompiled = layer2Data
+    .map((network) => {
+      return {
+        ...network,
+        txCosts: growThePieData.dailyTxCosts[network.growthepieID],
+        tvl: l2beatData.data.projects[network.l2beatID].tvl.breakdown.total,
+        networkMaturity: networkMaturity(
+          l2beatData.data.projects[network.l2beatID]
+        ),
+        activeAddresses: growThePieData.activeAddresses[network.growthepieID],
+        blockspaceData:
+          (growThePieBlockspaceData || {})[network.growthepieID] || null,
+        launchDate:
+          (growThePieMasterData?.launchDates || {})[
+            network.growthepieID.replace(/_/g, "-")
+          ] || null,
+        walletsSupported: walletsData
+          .filter((wallet) =>
+            wallet.supported_chains.includes(network.chain_name)
+          )
+          .map((wallet) => wallet.name),
+        walletsSupportedCount: `${
+          walletsData.filter((wallet) =>
+            wallet.supported_chains.includes(network.chain_name)
+          ).length
+        }/${walletsData.length}`,
+      }
+    })
+    .sort((a, b) => {
+      const maturityOrder = {
+        robust: 4,
+        maturing: 3,
+        developing: 2,
+        emerging: 1,
+      }
+
+      const maturityDiff =
+        maturityOrder[b.networkMaturity] - maturityOrder[a.networkMaturity]
+
+      if (maturityDiff === 0) {
+        return (b.tvl || 0) - (a.tvl || 0)
+      }
+
+      return maturityDiff
+    })
+
+  return {
+    props: {
+      ...(await serverSideTranslations(locale!, requiredNamespaces)),
+      contentNotTranslated,
+      lastDeployLocaleTimestamp,
+      locale,
+      layer2Data: layer2DataCompiled,
+      mainnetData: {
+        ...ethereumNetworkData,
+        txCosts: growThePieData.dailyTxCosts.ethereum,
+        tvl: "value" in ethereumTVLData ? ethereumTVLData.value : 0,
+      },
+    },
+  }
+}) satisfies GetStaticProps<BasePageProps>
+
+const Layer2Networks = ({ layer2Data, locale, mainnetData }) => {
+  const { pathname } = useRouter()
+
+  const heroProps: ContentHeroProps = {
+    breadcrumbs: { slug: pathname, startDepth: 1 },
+    heroImg: "/images/layer-2/learn-hero.png",
+    blurDataURL: "/images/layer-2/learn-hero.png",
+    title: "Choose network",
```

Comment:
"Choose network" feels restrictive to me as the title here... feel like you choose one and that's all, but I feel like "Explore networks" fits what we're trying to present better... @konopkja Any thoughts on this from SEO perspective?

Reply:
fine with explore